test/libsolidity/smtCheckerTests/literalSuffixes/division_by_zero_function_single_argument.sol

```diff
@@ -0,0 +1,8 @@
+function suffix(uint x) pure suffix returns (uint) { return 1000000 / x; }
+contract C {
+    uint y = 0 suffix;
+}
+// ====
+// SMTEngine: all
+// ----
+// Warning 4281: (60-71): CHC: Division by zero happens here.
```

Comment:
The new warning is not reported though

Reply:
Ah ok, new warning is only for two arguments. Is the suffix function actually properly supported then?